daos/mainnet/genesisalpha.json

```diff
@@ -4,5 +4,8 @@
   "DAOToken": "0xf88bfEab7217376F33db368A3d3DC11B1433bDFb",
   "Reputation": "0x30412F4c06343d65C06B5a0Bba6821cf37E605fa",
   "Controller": "0x211b9Bd0bCCACa64fB1d43093C75269bb84B3Be6",
+  "Schemes": {
+    "StandardBounties": "0x0a2Aca6C100BC39bee4C6643a3A8e061F06a624f"
```

Comment:
Please rename that to `GenericScheme` .
You can add an alias though . which will be called  `StandardBounties` . see reference https://github.com/daostack/subgraph/blob/master/daos/rinkeby/nectardao.json#L10

Reply:
Updated 77601d2